version.mdx

```diff
@@ -0,0 +1,111 @@
+---
+{
+    "1.0.0": {
+        "firebase_core": "2.30.0",
+        "firebase_auth": "4.0.0",
+        "cloud_firestore": "3.0.0",
+        "firebase_database": "10.0.0",
+        "firebase_storage": "10.0.0",
+        "firebase_messaging": "11.0.0",
+        "firebase_crashlytics": "3.0.0",
+        "firebase_performance": "2.0.0",
+        "firebase_remote_config": "4.0.0",
+        "firebase_analytics": "10.0.0",
+        "firebase_in_app_messaging": "1.0.0",
+        "firebase_app_check": "1.0.0",
+        "firebase_ml_vision": "1.0.0"
+    },
+    "1.1.0": {
+        "firebase_core": "2.30.0",
+        "firebase_auth": "4.0.0",
+        "cloud_firestore": "3.0.0",
+        "firebase_database": "10.0.0",
+        "firebase_storage": "10.0.0",
+        "firebase_messaging": "11.0.0",
+        "firebase_crashlytics": "3.0.0",
+        "firebase_performance": "2.0.0",
+        "firebase_remote_config": "4.0.0",
+        "firebase_analytics": "10.0.0",
+        "firebase_in_app_messaging": "1.0.0",
+        "firebase_app_check": "1.0.0",
+        "firebase_ml_vision": "1.0.0"
+    }
+}
+---
+
+# FlutterFire Compatible Versions
+
+This document is listing all the compatible versions of the FlutterFire plugins. This document is updated whenever a new version of the FlutterFire plugins is released.
+
+# Versions
+
+## [Flutter BoM 1.1.0 (2024-04-17)](https://github.com/firebase/flutterfire/blob/master/CHANGELOG.md#2024-04-16)
+
+Install this version using FlutterFire CLI
+
+```bash
+flutterfire install 1.1.0
+```
+
+### Included Native Firebase SDK Versions
+
+| Firebase SDK | Version | Link |
```

Comment:
Should we include this information in the JSON blurb at the top of the page too?

Reply:
I can add this to the JSON as well, in case it needs to be parsed by something else. Since we were not going to use it directly for the install command, didn't feel it was necessary, but can always be useful 🚀

Reply:
It has been added